search/search-domain/src/main/java/io/camunda/search/entities/DecisionRequirementsEntity.java

```diff
@@ -12,7 +12,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public record DecisionRequirementsEntity(
     String tenantId,
```

Comment:
💭  tenantId as first property makes it look like this is the PK. Is it maybe useful to change it that the PK is the first argument?